kymatio/scattering2d/frontend/torch_f.py

```diff
@@ -0,0 +1,178 @@
+        The scattering transform computes two wavelet transform followed
+        by modulus non-linearity.
+        It can be summarized as::
+
+            S_J x = [S_J^0 x, S_J^1 x, S_J^2 x]
+
+        where::
+
+            S_J^0 x = x * phi_J
+            S_J^1 x = [|x * psi^1_lambda| * phi_J]_lambda
+            S_J^2 x = [||x * psi^1_lambda| * psi^2_mu| * phi_J]_{lambda, mu}
+
+        where * denotes the convolution (in space), phi_J is a low pass
+        filter, psi^1_lambda is a family of band pass
+        filters and psi^2_mu is another family of band pass filters.
+        Only Morlet filters are used in this implementation.
+        Convolutions are efficiently performed in the Fourier domain
+        with this implementation.
+
+        Example
+        -------
+            # 1) Define a Scattering object as:
+            s = Scattering2D(J, shape=(M, N))
+            #    where (M, N) are the image sizes and 2**J the scale of the scattering
+            # 2) Forward on an input Tensor x of shape B x M x N,
+            #     where B is the batch size.
+            result_s = s(x)
+
+        Parameters
+        ----------
+        J : int
+            logscale of the scattering
+        shape : tuple of int
+            spatial support (M, N) of the input
+        L : int, optional
+            number of angles used for the wavelet transform
+        max_order : int, optional
+            The maximum order of scattering coefficients to compute. Must be either
+            `1` or `2`. Defaults to `2`.
+        pre_pad : boolean, optional
+            controls the padding: if set to False, a symmetric padding is applied
+            on the signal. If set to true, the software will assume the signal was
+            padded externally.
+
+        Attributes
+        ----------
+        J : int
+            logscale of the scattering
+        shape : tuple of int
+            spatial support (M, N) of the input
+        L : int, optional
+            number of angles used for the wavelet transform
+        max_order : int, optional
+            The maximum order of scattering coefficients to compute.
+            Must be either equal to `1` or `2`. Defaults to `2`.
+        pre_pad : boolean
+            controls the padding
+        Psi : dictionary
+            containing the wavelets filters at all resolutions. See
+            filter_bank.filter_bank for an exact description.
+        Phi : dictionary
+            containing the low-pass filters at all resolutions. See
+            filter_bank.filter_bank for an exact description.
+        M_padded, N_padded : int
+             spatial support of the padded input
+
+        Notes
+        -----
+        The design of the filters is optimized for the value L = 8
+
+        pre_pad is particularly useful when doing crops of a bigger
+         image because the padding is then extremely accurate. Defaults
+         to False.
```

Comment:
Default shouldn't be here but in the documentation of the constructor arguments.